clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -70,7 +70,7 @@ def describe_clusters():
             # The key isn't to do with clustering, skip it
             continue
 
-        if site != "":
+        if site != "" and remote_site != "":
```

Comment:
I'd like to stress that I really don't care, but wouldn't `if remote_site != "":` suffice?

If there's a remote site but the local site's name is blank, we have much bigger things to worry about than the string looking a bit odd.